README.MD

```
@@ -129,31 +129,33 @@ You can either:
+
+## Contributing
+
+We welcome contributions! See the [Contributing Guide](https://reloaded-project.github.io/reloaded-templates-rust/manual/#contributing) for details.
 
 ## License
 
-Licensed under [GPL v3 (with Reloaded FAQ)](./LICENSE).  
-[Learn more about Reloaded's general choice of licensing for projects.][reloaded-license].  
+Licensed under [GPL v3 (with Reloaded FAQ)](./LICENSE).
 
 [codecov]: https://about.codecov.io/
```

markdownlint-cli2 (0.18.1)

Link and image reference definitions should be needed
Unused link or image reference definition: "codecov"

(MD053, link-image-reference-definitions)